### ISSUE TITLE
Please update page with all 15 current activity types

### ISSUE THREAD

**Issue by v-kydela** [MicrosoftDocs/bot-docs#1335]
The types can be seen in the .NET SDK, the Swagger schema file, etc. They are message, contactRelationUpdate, conversationUpdate, typing, endOfConversation, event, invoke, deleteUserData, messageUpdate, messageDelete, installationUpdate, messageReaction, suggestion, trace, and handoff.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 79e3302e-8a46-f8f7-0b10-055a96e3a295
* Version Independent ID: 61bf86f6-271f-6172-262a-5e7a73233c88
* Content: [Activities overview - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-activities?view=azure-bot-service-4.0)
* Content Source: [articles/rest-api/bot-framework-rest-connector-activities.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/rest-api/bot-framework-rest-connector-activities.md)
* Service: **bot-service**
* GitHub Login: @RobStand
* Microsoft Alias: **kamrani**

**Comment by JonathanFingold** [comment]
See **bot-docs-pr** [PR 1698](https://github.com/MicrosoftDocs/bot-docs-pr/pull/1698) for more context.

**Comment by grantborthwick** [comment]
@JonathanFingold, your link 404s. I don't see a repo called bot-docs-pr under Microsoft Docs unless it is private. Under https://github.com/MicrosoftDocs/bot-docs/pulls, the most recent PR number is 1322.

**Comment by v-kydela** [comment]
@grantborthwick - It's a private repo.

**Comment by emgrol** [comment]
Hi @v-kydela, instead of reduplicating content @Kaiqb suggested that we just redirect that page in the TOC to [this page](https://github.com/Microsoft/botframework-sdk/blob/master/specs/botframework-activity/botframework-activity.md) that describes all of the activity types. The [PR](https://github.com/MicrosoftDocs/bot-docs-pr/pull/1703) is now updated to reflect that change, and the updates will go live this afternoon.

**Comment by v-kydela** [comment]
Thank you @emgrol 

I'm not sure I agree with the idea of relying on the activity spec page as a sort of schema document. That seems to be happening a lot and I don't know how that trend started. It's certainly a well-written document and it should be easy to find, but it can be hard to follow if a developer just wants to see something technical like all 15 activity types. There's no specific anchor on the page where the activity types are listed, so all we can do is link the reader to the whole page and expect them to work it out on their own, which isn't very user-friendly. And I notice that deleteUserData is missing from the spec, so if we replace the activities overview doc with the activity spec then we'll lose the deleteUserData documentation.

**Comment by emgrol** [comment]
I think these are good points @v-kydela. Do you have any suggestions/ideas @kaiqb?

**Comment by emgrol** [comment]
I talked with @Kaiqb about this again today and he said we should just reference the schema docs. 
Closing this now that the [PR](https://github.com/MicrosoftDocs/bot-docs-pr/pull/1703) has been merged, changes will go live later today.
